lxd/patches.go

```diff
@@ -1445,4 +1447,40 @@ DELETE FROM storage_pools_config WHERE key = "powerflex.sdt"
 	return err
 }
 
+// patchOIDCGroupsClaimScope adds the contents of oidc.groups.claim to the new configuration for oidc.scopes if present.
+// The oidc.groups.claim value was initially added to scopes but shouldn't have been. This patch will allow users with
+// working identity provider group mappings to continue using them by continuing to request the claim as an additional
+// scope.
+func patchOIDCGroupsClaimScope(_ string, d *Daemon) error {
```

Comment:
@markylaing this function will be run on each cluster member, at different times.

I notice that it is loading the cluster config from the DB inside the function, and then modifying the global config.

But is it safe when potentially this function is run again on a different cluster member after the global config has been modified by this function at an earlier stage?

Reply:
Ahh of course :facepalm: will fix

Reply:
Fixed. Thanks for catching :)